src/studio/src/designer/frontend/shared/components/molecules/AltinnInformationCardForChildren.module.css

```diff
@@ -0,0 +1,76 @@
+.container {
+  height: 80vh;
+  justify-content: center;
+  align-content: center;
+  flex-direction: column;
+  display: flex;
+}
+.paper {
+  background: white;
+  padding: 78px 45px 100px;
+  max-width: 944px;
+  height: 446px;
+  display: flex;
+  flex-direction: row;
+}
+/*
+.paper-xl {
+  padding-left: 99px;
+  padding-top: 92px;
+  padding-bottom: 87px;
+  max-width: 1088px;
+  height: 446px;
+}
+.paper-lg {
+  padding-left: 101px;
+  padding-top: 82px;
+  padding-bottom: 97px;
+  max-width: 1088px;
+  height: 446px;
+}
+.paper-md {
+  padding-left: 45px;
+  padding-top: 78px;
+  padding-bottom: 100px;
+  max-width: 944px;
+  height: 446px;
+}
+.paper-sm {
+  padding-left: 56px;
+  padding-top: 68px;
+  padding-bottom: 97px;
+  max-width: 554px;
+  height: 623px;
+}
+*/
```

Comment:
Simply delete this :)

Reply:
These was mediaqueries... I have now created them back to media-queries.